social.css

```diff
@@ -0,0 +1,120 @@
+.wrap-social{
+    width: var(--main-container-width);
+    background-color: rgb(248, 239, 239);
+}
+
+.social {
+    width: var(--main-section-width);
+    margin: var(--main-section-margin);
+    display: flex;
+}
+
+.socail__text {
+    flex-grow: 1;
+    font-size: 1.5rem;
+}
+
+.social__text p {
+    font-family: var(--second-font-family);
+    font-size: 1.125rem;
+}
+
+.social__networks {
+    flex-grow: 1;
+    height: 8.75rem;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+}
+
+.wrap-main1 {
+    padding-top: 6.88rem;
+    width: var(--main-container-width);
+    background-color: rgb(255, 255, 255);
+    padding-bottom: 6.25rem;
+}
+
+.main1 {
+    width: var(--main-section-width);
+    margin: var(--main-section-margin);
+}
+
+.main1__buttons {
+    display: flex;
+    justify-content: flex-start;
+}
+
+.main1__buttons button {
+    width: 11.5rem;
+    border: none;
+    background-color: rgb(255, 255, 255);
+    margin: 0;
+    padding: 0.63rem 0;
+    font-family: var(--main-font-family);
+    font-size: 14px;
+}
+
+.main1__buttons > .tab1 {
+    z-index: 2;
+    width: 12.56rem;
+    border-right: 1px solid rgb(198, 203, 205);
```

Comment:
refactor this

Reply:
![image](https://user-images.githubusercontent.com/70269636/155888761-d13196e9-ace6-4e43-b82e-2e6500bfd313.png)